packages/cli/src/commands/generate/scaffold/templates/lib/formatters.tsx.template

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+import humanize from 'humanize-string'
```

Comment:
Since the import is explicitly in the users's project, the dependency should probably also be explicit in the project's package.json, right? Should I make the scaffold generator run `yarn add`?

Reply:
This is a weird one... maybe we should add it as a dependency on rwjs/web? It feels like a hammer, but atleast its "backwards" compatible.

The reason it may not be an issue is because webpack will bundle dependencies anyway. So the other option is to not do anything, since its part of the cli dependencies anyway.